src/Resolver/ValueInjection.php

```diff
@@ -68,15 +74,36 @@ public function export() : string
      */
     public function isExportable() : bool
     {
-        if (is_scalar($this->value) || ($this->value === null)) {
+        return $this->isExportableRecursive($this->value);
+    }
+
+    /**
+     * Check if the provided value is exportable.
+     * For arrays it uses recursion.
+     *
+     * @param mixed $value
+     * @return bool
```

Comment:
The `@return` annotation does not provide any additional information and should be omitted